opencog/atomspace/atom_types.script

```diff
@@ -205,8 +211,8 @@ UNQUOTE_LINK <- ORDERED_LINK
 // Assorted PLN link types.
 
 // Logical quantifiers
-FORALL_LINK <- SCOPE_LINK "ForAllLink"
-EXISTS_LINK <- SCOPE_LINK
+FORALL_LINK <- LAMBDA_LINK "ForAllLink"
```

Comment:
well, we had this conversation, and this is clearly wrong, so I am confused about why you are changing this,,,